cpanfile

```diff
@@ -0,0 +1,15 @@
+requires "Class::Method::Modifiers" => "0";
+requires "Email::Stuffer" => "0";
+requires "Mojo::UserAgent" => "0";
+requires "Test::Most" => "0";
+requires "strict" => "0";
```

Comment:
Since `strict`+`warnings` are core (and always likely to be), they can be removed

Reply:
Done